src/celeritas/em/interactor/CombinedBremInteractor.hh

```diff
@@ -54,7 +54,7 @@ class CombinedBremInteractor
   public:
     // Construct with shared and state data
     inline CELER_FUNCTION
-    CombinedBremInteractor(CombinedBremRef const& shared,
+    CombinedBremInteractor(NativeCRef<CombinedBremData> const& shared,
```

Comment:
Since the combined brem executor currently only works for the single element case (is this because we never ported the per-element xs?) should we delete the combined brem interactor?

Reply:
Right, it's because we don't have micro xs for the combined model. From our past discussions it sounded like we didn't see much benefit from the combined model when we tested very early on. We've talked about taking another look once the geometry's sped up and the physics is a more significant fraction of the time, and possibly also testing combined interactors for the muon models. I think we should either build the combined cross sections or remove the model, and given that this is pretty low on our list of performance concerns (and that we could always revert it later) I'm more in favor of the latter.

Reply:
Agreed!